shell.nix

```diff
@@ -4,6 +4,7 @@ pkgs.mkShell {
   buildInputs = with pkgs; [
     figlet
     stdenv
+    cacert
```

Comment:
General observation. Unless things need a specific order prefer alphabetical order. Having an order makes it easier to find things as lists grow.
If a specific order is required due to dependencies, adding a comment saves everyone including future you.